app/views/rsvp.erb

```diff
@@ -1,14 +1,34 @@
 <div class="offset" id="rsvp"></div>
+
 <% if session['access_token'] %>
-  <form class="form-horizontal" method="POST" action="/rsvp">
-  <fieldset>
+
+<% # to validate if the user in either of me and Nikhil's friend list
```

Comment:
Please don't include business logic in the view template.
I can explain how you can share variables from the controller(route) to the view.